manifests/2.5.0/opensearch-2.5.0.yml

```diff
@@ -163,3 +163,12 @@ components:
       - windows
     checks:
       - gradle:properties:version
+  - name: asynchronous-search
+    repository: https://github.com/opensearch-project/asynchronous-search.git
+    ref: 2.5
```

Comment:
Should we add tags or branch in ref ? otherwise approving it

Reply:
Branch as tags are cut after the release. @Arpit-Bandejiya please make `2.5` a string by enclosing it in quotes. Thanks!

Reply:
Done @gaiksaya, please check.

Reply:
Looks like there is a merge conflict. Can you rebase and fix it?
Thanks!

Reply:
fixed!